docker-compose.yml

```diff
@@ -174,6 +173,8 @@ services:
       - _APP_MAINTENANCE_RETENTION_USAGE_HOURLY
       - _APP_SMS_PROVIDER
       - _APP_SMS_FROM
+      - _APP_GRAPHQL_MAX_COMPLEXITY
+      - _APP_GRAPHQL_MAX_DEPTH
```

Comment:
Should we also include `_APP_GRAPHQL_MAX_BATCH_SIZE`?

Reply:
I think we should, worst case we can revert later.